ui/raidboss/data/06-ew/raid/p1s.txt

```diff
@@ -0,0 +1,106 @@
+### P1S: Asphodelos: The First Circle (Savage)
+#
+# -it 'Erichthonios' -oc 'Erichthonios' -p "6629:14.1"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+# First flail set
+0.0 "--sync--" sync /Engage!/ window 0,1
+9.4 "--sync--" sync / 14:[^:]*:Erichthonios:6629:/ window 11,9
+14.1 "Heavy Hand" sync / 1[56]:[^:]*:Erichthonios:6629:/
+18.2 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
```

Comment:
I believe the boss teleports middle with this spell, perhaps --middle-- instead of --sync--?

Reply:
Sorry, it should have read to replace all the "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/ with "--middle--" sync / 1[56]:[^:]*:Erichthonios:65F5:/ . Thanks for the update.